opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/__init__.py

```diff
@@ -196,7 +197,29 @@ def create_observable_counter(
             self._instrument_id_instrument[instrument_id] = instrument
             return instrument
 
-    def create_histogram(self, name, unit="", description="") -> APIHistogram:
+    def create_histogram(
+        self,
+        name: str,
+        unit: str = "",
+        description: str = "",
+        advisory: Optional[MetricsInstrumentAdvisory] = None,
+    ) -> APIHistogram:
+        if advisory is not None:
+            raise_error = False
+            try:
+                boundaries = advisory["explicit_bucket_boundaries"]
+                raise_error = not (
+                    boundaries
+                    and all(isinstance(e, (int, float)) for e in boundaries)
+                )
+            except (KeyError, TypeError):
+                raise_error = True
+
+            if raise_error:
+                raise ValueError(
+                    "Advisory must be a dict with explicit_bucket_boundaries key containing a sequence of numbers"
+                )
```

Comment:
Copied [from the spec](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.40.0/specification/metrics/sdk.md#instrument-advisory-parameters:~:text=When%20a%20Meter%20creates%20an%20instrument%2C%20it%20SHOULD%20validate%20the%20instrument%20advisory%20parameters.%20If%20an%20advisory%20parameter%20is%20not%20valid%2C%20the%20Meter%20SHOULD%20emit%20an%20error%20notifying%20the%20user%20and%20proceed%20as%20if%20the%20parameter%20was%20not%20provided.), emphasis mine

> When a Meter creates an instrument, it SHOULD validate the instrument advisory parameters. If an advisory parameter is not valid, the Meter SHOULD emit an error notifying the user **and proceed as if the parameter was not provided.**

Reply:
Corrected, thanks!